protocols/migration/migration_reports_4_0_0_to_reports_5_0_0.py

```diff
@@ -86,7 +86,7 @@ def migrate_interpreted_genome_rd(self, old_instance, assembly, interpretation_r
         new_instance.interpretationService = old_instance.companyName
 
         # reportUri has changed to reportUrl
-        new_instance.reportUrl = old_instance.reportUri
+        new_instance.reportUrl = old_instance.reportUrl
```

Comment:
if the field name, `reportUrl`, does not change, then can we just delete this line and the comment ?

Reply:
indeed!